privaterelay/templates/includes/banners.html

```diff
@@ -0,0 +1,20 @@
+<div class="dashboard-banners flx invisible hide-750">
+  <div class="banner-gradient-bg">
+    <div class="download-fx-banner banner-content flx flx-row hidden">
+      <div class="banner-fx-logo banner-logo"></div>
+      <div class="banner-copy flx flx-col">
+        <p class="banner-hl ff-Met">Relay is even better in Firefox</p>
+        <p class="banner-sub">The Relay extension for Firefox Browser makes creating aliases even easier.</p>
+        <a class="banner-link ff-Met" href="https://www.mozilla.org/firefox/new/?utm_source=fx-relay&utm_medium=banner&utm_campaign=download-fx"" target="_blank" rel="noopener noreferrer">Get Firefox</a>
+      </div>
+    </div>
+    <div class="install-addon-banner banner-content flx flx-row hidden">
+      <div class="banner-relay-logo banner-logo"></div>
+      <div class="banner-copy flx flx-col">
+        <p class="banner-hl ff-Met">Looks like you haven't installed the add-on... yet!</p>
+        <p class="banner-sub">The Relay extension for Firefox Browser makes creating aliases even easier.</p>
+        <a class="banner-link ff-Met" href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay&utm_medium=banner&utm_campaign=install-addon" target="_blank" rel="noopener noreferrer">Install the Firefox Relay addon</a>
```

Comment:
nit: you say "addon" here, but "add-on" above on L14. We should be consistent, but I think the preference is hyphenated (per https://addons.mozilla.org/en-US/firefox/ labels).